pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -87,7 +87,7 @@ def default_pypeit_par(cls):
         par['reduce']['skysub']['global_sky_std']  = False    # Do not perform global sky subtraction for standard stars
         par['reduce']['skysub']['no_poly'] = True             # Do not use polynomial degree of freedom for global skysub
         par['reduce']['extraction']['model_full_slit'] = True  # local sky subtraction operates on entire slit
-
+        par['reduce']['findobj']['maxnumber'] = 1  # Slit is narrow so allow one object per order
```

Comment:
Do we really want this set as a default?
  Would seem better to advise people to consider
  adding this as needed.